src/button-dropdown/category-elements/expandable-category-element.tsx

```diff
@@ -67,6 +71,19 @@ const ExpandableCategoryElement = ({
       ref={triggerRef}
       {...getMenuItemProps({ parent: true, expanded, disabled })}
       {...(isDisabledWithReason ? targetProps : {})}
+      {...getAnalyticsMetadataAttribute(
+        disabled
+          ? {}
+          : ({
+              action: 'expand',
+              detail: {
+                position: position || '0',
+                label: '',
+                id: item.id || '',
+                expanded: `${!expanded}`,
```

Comment:
Why do we negate here? Are we including the previous value in the detail or the current?